### PR TITLE
Add asp.net web server to solutions

### PR DESCRIPTION
**Pull request by jxyang**
Signed-off-by: Xuejun Yang <xuejya@microsoft.com>

### CODE REVIEW

File: solutions/aspnet/Program.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        {
+            CreateHostBuilder(args).Build().Run();
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureWebHostDefaults(webBuilder =>
+                {
+                    webBuilder.UseStartup<Startup>();
+										//webBuilder.UseKestrel();
```
Comment (vtikoo) [inline solutions/aspnet/Program.cs:24]: nitpick: Indentation

File: solutions/aspnet/Makefile [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+APPNAME=webapp
+
+#ifdef STRACE
+#OPTS += --strace
+#endif
+
+all: appdir private.pem
+
+run: appdir private.pem
+	test -f server.pid && kill -9 `cat server.pid` || true
```
Comment (vtikoo) [inline solutions/aspnet/Makefile:14]: Maybe add a comment here explaining what this line achieves?

**Review by vtikoo — APPROVED** [review]
Thanks for making this possible! 
LGTM. Minor comments about formatting and documentation.

**Comment by jxyang** [comment]
Thanks Vikas. I have addressed all your comments.

**Pull request merged**